docs/current/sdk/go/959738-get-started.md

```diff
@@ -62,9 +66,10 @@ func main() {
 func build(repoUrl string) error {
   fmt.Printf("Building %s\n", repoUrl)
 
-  // 1. Get a context
+  // get context
```

Comment:
nit: I think we can actually just remove this comment entirely; go developers know what's happening here

Reply:
Done